app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] == "cp" and len(parts) == 3 and parts[1] != parts[2]:
+        source_file, destination_file = parts[1], parts[2]
```

Comment:
you still use indexing, first make unpacking like this `cp, source_file, destination_file = parts` and then make another checking with variable names